src/NLsolve.jl

```diff
@@ -31,16 +29,20 @@ end
 show(io::IO, e::IsFiniteException) = print(io,
   "During the resolution of the non-linear system, the evaluation" *
   " of the following equation(s) resulted in a non-finite number: $(e.indices)")
-
-include("differentiable_functions.jl")
-include("solver_state_results.jl")
-include("nlsolve_func_defs.jl")
-include("mcp_func_defs.jl")
-include("utils.jl")
-include("newton.jl")
-include("trust_region.jl")
-include("anderson.jl")
-include("autodiff.jl")
-include("mcp.jl")
+df_path = "differentiable_vectors/"
+include(df_path*"differentiable_functions.jl")
```

Comment:
use `joinpath` instead

Reply:
I rather just have `inlucde("path/file.jl")`. That way static analyzers (like the VSCode plugin) can follow the include. It is not powerful enough to "constant propagate" `joinpaths`.